entries/u/umbrella.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Umbrella": {
+    "domain": "umbrella.com",
```

```suggestion
    "domain": "umbrella.com",
    "img": "cisco.com.svg",
```
as the icon already exists in the directory, we can just link to that one instead of uploading a new :wink: